content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,101 @@
+`JENKINS_HOME` has a reasonably obvious directory structure that looks like the following:
+
+[width="100%",cols="100%",]
+|===
+a|
+....
+JENKINS_HOME
+    +- config.xml     (jenkins root configuration)
+    +- *.xml          (other site-wide configuration files)
+    +- userContent    (files in this directory will be served under your http://server/userContent/)
+    +- fingerprints   (stores fingerprint records)
+    +- nodes          (slave configurations)
+    +- plugins        (stores plugins)
+    +- secrets        (secretes needed when migrating credentials to other servers)
+    +- workspace (working directory for the version control system)
+        +- [JOBNAME] (sub directory for each job)
+    +- jobs
+        +- [JOBNAME]      (sub directory for each job)
+            +- config.xml     (job configuration file)
+            +- latest         (symbolic link to the last successful build)
+            +- builds
+                +- [BUILD_ID]     (for each build)
+                    +- build.xml      (build result summary)
+                    +- log            (log file)
+                    +- changelog.xml  (change log)
+....
+|===
+
+=== Backup and restore
+All the settings, build logs, and artifact archives are stored under the `JENKINS_HOME` directory.
+Simply archive this directory to make a backup.
+Similarly, restoring the data is just replacing the contents of the `JENKINS_HOME` directory from a backup.
+Backups can be taken without stopping the server, but when you restore, please stop the server before doing so.
+
+=== Moving/copying/renaming jobs
+You can:
+
+. Move a job from one installation of Jenkins to another by simply copying the corresponding job directory.
+. Make a copy of an existing job by making a clone of a job directory by a different name.
+. Rename an existing job by renaming a directory. Note that the if you change a job name you will need to change any other job that tries to call the renamed job.
+
+Those operations can be done even when Jenkins is running. For changes like these to take effect, you have to click "reload config" to force Jenkins to reload configuration from the disk.
+
+==== Batch renaming jobs
+Replacing spaces in job names with underscores.
+[width="100%",cols="100%",]
+|===
+a|
+....
+$ rename 's/\s/_/g' *
+....
+|===
+
+=== Archive unused jobs
+Sometimes you want to remove a job from Jenkins but do so in such a way that you can resurrect it later, if the need arises.
+You can do this by going to `$JENKINS_HOME` and creating an archive of the job directory.
+The following command illustrates how to archive a job 'xyz' and remove it.
+[width="100%",cols="100%",]
+|===
+a|
+....
+$ cd $JENKINS_HOME/jobs
+$ tar czf xyz.tgz xyz
+# go to Jenkins GUI "Manage Jenkins" page and "Reload Configuration from Disk"
+....
+|===
+As long as you are not building the xyz project while you create an archive, you can do this operation without taking Jenkins offline.
```

Comment:
```suggestion
As long as you are not building the
You can do this operation without taking Jenkins offline
although you should not build the xyz project while you create an archive.
```

Reply:
Need more detail.

Reply:
"As long as you are not building the" Is there a word missing?